k3s-leader/k3s-leader.md

```diff
@@ -0,0 +1,102 @@
+﻿![K3S logo](https://k3s.io/images/logo-k3s.svg "K3S Logo")
+
+## What is K3S?
+
+---
+
+Lightweight Kubernetes
+The certified Kubernetes distribution built for IoT & Edge computing
+
+## Features
+
+---
+
+-Perfect for Edge
+K3s is a highly available, certified Kubernetes distribution designed for production workloads in unattended, resource-constrained, remote locations or inside IoT appliances.
+
+-Simplified & Secure
+K3s is packaged as a single <50MB binary that reduces the dependencies and steps needed to install, run and auto-update a production Kubernetes cluster.
+
+-Optimized for ARM
+Both ARM64 and ARMv7 are supported with binaries and multiarch images available for both. K3s works great from something as small as a Raspberry Pi to an AWS a1.4xlarge 32GiB server.
+
+## Installation
+
+---
+
+1. Select the devices to which K3S-Follower will be installed from 'Devices' page.
+
+2. Navigate to the 'App Marketplace' tab and select the 'K3S-Follower' application.
+
+3. The 'Install Now' button should now appear near the top of the screen. Select this button.
+
+4. Give your installation a name and fill all the parameters. Click 'Install K3S-Follower' in the bottom right corner.
+
+## Required Parameters
+
+---
+
+**K3S_TOKEN**
+Secret token used for joining new nodes
+
+**IP_ADDRESS**
```

Comment:
IP address of an existing leader node. Leave blank if installing the first leader